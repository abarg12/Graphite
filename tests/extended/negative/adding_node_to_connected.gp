node<int> n;
graph<int, connected> g;
g.addNode(n);
printf("success");