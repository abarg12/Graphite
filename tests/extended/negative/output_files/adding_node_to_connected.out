Fatal error: exception Failure("Other dot methods are not currently implemented")
